bin/copyStudyUtils.js

```diff
@@ -3,13 +3,18 @@
 const path = require("path");
 const fs = require("fs-extra");
 
-const files = ["study/api.js", "study/schema.json"];
+const files = [
+  "study/api.js",
+  "study/schema.json",
+  "testingOverrides/api.js",
```

Comment:
People are leaving in the magic prefs even during deployment.  #211  .  I am not convinced that doing this as a separate api is better.

-1 

I would prefer a discussion bug to get the list, name the prefs, and discuss how to get them into the product.

See #208 

- Should they have addon specific names?  
- Should they persist?

Reply:
Moved discussion to https://github.com/mozilla/shield-studies-addon-utils/issues/212